src/com/projectkorra/projectkorra/Element.java

```diff
@@ -72,8 +72,8 @@ public String getBend() {
 	 * 
 	 * @param name Name of the new Element.
 	 */
-	public Element(String name) {
-		this(name, ElementType.BENDING, ProjectKorra.plugin);
+	public Element(String name, String color, String prefix) {

```

Comment:
We would probably want this to be `ChatColor` color and `String` prefix to make it a little less difficult for typos.